src/status_im/ui/screens/communities/community_options_bottom_sheet.cljs

```diff
@@ -0,0 +1,90 @@
+(ns status-im.ui.screens.communities.community-options-bottom-sheet
+  (:require  [status-im.i18n.i18n :as i18n]
+             [status-im.ui.components.react :as react]
+             [quo2.components.markdown.text :as text]
+             [quo2.components.buttons.button :as button]
+             [quo2.components.drawers.action-drawers :as action-drawers]
+             [quo2.components.tags.context-tags :as context-tags]
+             [status-im.react-native.resources :as resources]
+             [status-im.utils.handlers :refer [<sub >evt]]
+             [status-im.communities.core :as communities]))
+
+(def not-joined-options [{:icon  :main-icons2/members
+                          :label (i18n/label :t/view-members)}
+                         {:icon :main-icons2/bullet-list
+                          :right-icon :main-icons2/chevron-right
+                          :label (i18n/label :t/view-community-rules)}
+                         {:icon :main-icons2/add-user
+                          :label  (i18n/label :t/invite-contacts)}
+                         {:icon :main-icons2/qr-code
+                          :label (i18n/label :t/show-qr)}
+                         {:icon :main-icons2/share-context
+                          :label (i18n/label :t/share-community)}])
+
+(def joined-options [{:icon :main-icons2/members
+                      :label (i18n/label :t/view-members)}
+                     {:icon :main-icons2/bullet-list
+                      :right-icon :main-icons2/chevron-right
+                      :label (i18n/label :t/view-community-rules)}
+                     {:icon :main-icons2/up-to-date
+                      :label  (i18n/label :t/mark-as-read)}
+                     {:icon :main-icons2/muted
+                      :label  (i18n/label :t/mute-community)
+                      :right-icon :main-icons2/chevron-right}
+                     {:icon :main-icons2/notifications
+                      :label  (i18n/label :t/community-notification-settings)
+                      :right-icon :main-icons2/chevron-right}
+                     {:icon :main-icons2/add-user
+                      :label  (i18n/label :t/invite-contacts)}
+                     {:icon :main-icons2/qr-code
+                      :label (i18n/label :t/show-qr)}
+                     {:icon :main-icons2/share-context
+                      :label (i18n/label :t/share-community)}])
+
+(defn leave-sheet [community]
+  [react/view {:style {:flex 1 :margin-left 20 :margin-right 20 :margin-bottom 20}}
+   [react/view {:style {:flex 1 :flex-direction :row :align-items :center :justify-content :space-between}}
```

Comment:
I think we agreed on using `rn` for consistency?

Reply:
yep, you're right, I'll update :)